ERCS/erc-7087.md

```diff
@@ -0,0 +1,163 @@
+---
+eip: 7087
+title: New features for Web3 URL in Auto Mode
```

Comment:
"New Features" doesn't differentiate this proposal enough. Imagine five years down the road, having a dozen "New Features" proposals. It'd be hard to tell them apart.

Reply:
Good point!